src/legacy/core_plugins/kibana/public/management/sections/objects/_objects.js

```diff
@@ -40,6 +40,7 @@ function updateObjectsTable($scope, $injector) {
   const $http = $injector.get('$http');
   const kbnUrl = $injector.get('kbnUrl');
   const config = $injector.get('config');
+  const confirmModalPromise = $injector.get('confirmModalPromise');
```

Comment:
Since we are already refactoring this, maybe we should be using the new core modal service that @joshdover is working on? #36057 

Sounds like it might be landing by the end of this week.

Reply:
can i open confirm modal with it yet ?

Reply:
Looks like no, I guess we'll just hold on this for now.